.github/workflows/test_on_push.yaml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-tests-ce:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
+    strategy:
+      matrix:
+        tarantool-version: ["1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+      fail-fast: false
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Install requirements for community
+        run: |
+          curl -L https://tarantool.io/installer.sh | sudo VER=${{ matrix.tarantool-version }} bash
```

Comment:
If we only need to run tests on Linux on GitHub owned runners on latest tarantool releases within a release series at the moment, it is better to use https://github.com/tarantool/setup-tarantool: it caches dependencies to speed up CI.

Reply:
If you aims to implement more powerful CI, I suggest to look at tarantool/smtp: https://github.com/tarantool/smtp/tree/master/.github/workflows

Still not powerful enough, though:

https://github.com/tarantool/smtp/issues/34
https://github.com/tarantool/smtp/issues/35
https://github.com/tarantool/smtp/issues/36